docs/aerodromes/classc/Brisbane.md

```diff
@@ -29,6 +29,74 @@ There are three SMC positions at Brisbane which are responsible for various part
   <figcaption>YBBN Maneuvering Area Responsibility</figcaption>
 </figure>
 
+## Helicopter Operations
+Brisbane has one helipad located within the general aviation apron, most rescue helicopters are based within the general aviation maintenance area and will depart from the **H2 taxiway** to save time when operating.
+
+#### Waypoints
+VFR and IFR helicopters are normally processed via two visual waypoints north and south of the field, when in VMC conditions. IFR helicopters in IMC conditions may conform to fixed wing ops and be processed via the BN (RADAR) SID from an appropriate runway, unless a visual departure is acceptable. These waypoints are used for **departures** and **arrivals**, depending on weather and traffic conditions.
```

Comment:
```suggestion
In VMC, all helicopters are normally processed via two visual waypoints (outlined below) positioned north and south of the field. Where VMC does not exist, IFR helicopters may conform to fixed wing ops and be processed via the **BN (RADAR) SID** from an appropriate runway, unless a visual departure is acceptable.
```

Reply:
Also would we be able to get a vatSys diagram showing these waypoints? We might be able to talk to the procedural separation acheived by using them as clearance limits, etc here as well

Reply:
Yeah I would be able to put that together, however VatSys doesn't include Manly Boat Harbour in the BN_VFR maps, only Bald Hills. 

![image](https://github.com/user-attachments/assets/abbec401-ed57-4bee-a8f6-5f0b719e9af4)